kythe/go/extractors/config/runextractor/README.md

```diff
@@ -30,13 +30,14 @@ Use:
 
 Because `runextractor` is invoked in a manner not conducive to cleanly passing
 commandline flags, there is non-trivial setup done with environment variables.
-When calling `runextractor`, here are the relevant environment variables:
+When calling `runextractor`, here are the required environment variables:
 
-* **KYTHE_CORPUS**: The corpus set for extracted files.
-* **KYTHE_ROOT_DIRECTORY**: The root path for file input to be extracted.
-* **KYTHE_OUTPUT_DIRECTORY**: The path for storing output.
+* **KYTHE_ROOT_DIRECTORY**: The absolute path for file input to be extracted.
```

Comment:
Absolute paths remove one kind of uncertainty, but replace it with another: Instead of not knowing what the paths are supposed to be relative to, the absolute path makes it unclear which part of the path belongs to the build tool and which is happenstance (e.g., I run in a directory named `/tmp/build/1ca039beefc0d3000/test/kythe/go/platform/foo/testdata/kythe/go/platform/input.go`, how do I know where the root of the repo is so I can generate paths?)

Of course, that doesn't mean we _can't_ use absolute paths, but it might wind up making more sense to stipulate that the current working directory when the tool starts up affects the interpretation of these values. (We might not have enough data to really define that yet, I'm just thinking ahead).

Reply:
Yea and there's another wrench in here which is docker-in-docker does funky things with paths set (the path might be an absolute path on a different image, or the root (or both!).

For now, absolute path, if only until we get rid of docker in docker.

But I will put a note here for the dind removal bit to reevaluate what to do here.